src/app/auth/reset-password/reset-password.page.ts

```
@@ -53,11 +53,7 @@ export class ResetPasswordPage {
 
     this.routerAuthService
       .sendResetPassword(email)
-      .pipe(
-        finalize(() => {
-          this.isLoading = false;
-        })
-      )
+      .pipe(tap(() => (this.isLoading = false)))
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)